### ISSUE TITLE
Shared Url between Pages and Sites

### ISSUE THREAD

**Issue by ajahangard** [oqtane/oqtane.framework#3149]
Hi,
I think there is a problem in validation of data while adding/editing pages and sites.
If there is a Site with path "Blog" one can add a Page with path "Blog" to the main site and set the parent to root.
In this case the address for both Page and Site will be for example: "http://localhost/Blog".
Now if one types the address in address bar the site will be shown and if clicks on the blog's menu item the page will be shown.
I think when adding and editing pages, sites should be checked so that no site with same path exists and vice versa.
thank you.

**Comment by sbwalker** [comment]
@ajahangard you are correct that if you create a micro-site at www.mysite.com/test and also create a page in www.mysite.com named "test" that the paths are essentially duplicates... and the micro-site will take precedence (site has higher priority than page). I do not disagree with including more validation... its more a matter of the feasibility.
